### ISSUE TITLE
How to set up p2pool?

### ISSUE THREAD

**Issue by ThunderX-HUB** [UnamSanctam/SilentCryptoMiner#1838]
I need help (again...)

**Comment by UnamSanctam** [comment]
There is a get started guide here: https://p2pool.io/#help. Essentially you need to set up a Monero node locally and after that connect your p2pool instance t that. And then after that you connect your miner to that.

**Comment by ThunderX-HUB** [comment]
What's happening?
![Screenshot_2023-09-30-07-09-23-13](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104974936/a32a8601-971d-40c6-b495-4f64a82f27ed)

**Comment by UnamSanctam** [comment]
Could either be too many rejected shares (those happen naturally) though there aren't a lot of them. Could also be too many submitted shares overall over a short period of time (in which case use a higher difficulty port). Otherwise you could have too many upstreams (connections to the pool) but it doesn't really seem like that either, but that's the most likely one. But that error/block you got usually happens with MoneroOcean.

**Comment by ThunderX-HUB** [comment]
> Could either be too many rejected shares (those happen naturally) though there aren't a lot of them. Could also be too many submitted shares overall over a short period of time (in which case use a higher difficulty port). Otherwise you could have too many upstreams (connections to the pool) but it doesn't really seem like that either, but that's the most likely one. But that error/block you got usually happens with MoneroOcean.

That's why I really want to use p2pool but is it more complicated than xmrig-proxy?

**Comment by UnamSanctam** [comment]
Not really, but you can use xmrig-proxy and p2pool at the same time, though just using p2pool would be easier. The only thing is that downloading the blockchain for the node can take some time.
